### PR TITLE
Fix typos in BreadCrumbBarPage sample

### LINKED ISSUE #1254: Typos in BreadcrumbBar sample
### Which version of the app?

WinUI 3 Gallery

### Description

Code sample for BreadcrumbBar example 2 has typos. The control name in XAML doesn't match the source code. Source code has missing ; and fails to compile.

### Screenshots

image

### Windows version

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by chingucoding**
## Description

Fixes #1254
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUIGallery/ControlPages/BreadcrumbBarPage.xaml.cs [hunk 1]
```diff
@@ -20,25 +20,38 @@ public BreadcrumbBarPage()
...
-                items.RemoveAt(i);
+                items.Add(folders[i]);
             }
         }
 
-        private void ResetSampleButton_Click(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
+        public class Folder
+        {
+            public string Name { get; set; }
+        }
```
Comment (karenbtlai) [inline WinUIGallery/ControlPages/BreadcrumbBarPage.xaml.cs:46]: This seems to be added by accident.

File: WinUIGallery/ControlPages/BreadcrumbBarPage.xaml.cs [hunk 2]
```diff
@@ -20,25 +20,38 @@ public BreadcrumbBarPage()
             this.InitializeComponent();
             BreadcrumbBar1.ItemsSource = new string[] { "Home", "Documents", "Design", "Northwind", "Images", "Folder1", "Folder2", "Folder3" };
 
-            BreadcrumbBar2.ItemsSource = new ObservableCollection<Folder>(folders);
+
+            BreadcrumbBar2.ItemsSource = new ObservableCollection<Folder>{
```
Comment (karenbtlai) [inline WinUIGallery/ControlPages/BreadcrumbBarPage.xaml.cs:24]: Why does is the Folder ObservableCollection declared here? Can we not reference the List that is declared at the top?

**Review by karenbtlai — CHANGES_REQUESTED** [review]
See comments.

**Comment by chingucoding** [comment]
Guess I forgot to remove my testing of the C# sample inside the project, should be fixed now.

**Review by karkarl — APPROVED** [review]

**Pull request merged**